packages/gatsby/gatsby-browser.js

```diff
@@ -34,7 +34,8 @@ exports.onClientEntry = function(_, pluginParams) {
       // eslint-disable-next-line no-undef
       dsn: __SENTRY_DSN__,
       ...pluginParams,
-      tracesSampleRate,
+      tracesSampleRate: pluginParams.tracesSampleRate,
```

Comment:
These 2 lines are redundant, as `...pluginParams` will include them implicitly

Reply:
Shoot, you're right - we're no longer modifying the value, so it doesn't need to be separate like it used to be. Fixed.